config.go

```diff
@@ -39,14 +39,14 @@ const (
 // Config represents the configuration for the command.
 type Config struct {
 	DataDir string `toml:"data-dir"`
-	Host    string `toml:"host"`
+	Host    string `toml:"bind"`
```

Comment:
Good call - I didn't think of that. I'm refactoring tools to try not to import Pilosa at all. Since it isn't open source yet, I say let's not let that stop us. If the pdk were using it, that would be a bigger issue (I haven't checked),  but since everything is vendored, changes shouldn't break anything.